prep-javascript-objects/main.js

```diff
@@ -0,0 +1,19 @@
+const person = {
+  firstName: 'Jay',
+  lastName: 'Ghil',
+  hobby: 'reading manga'
+};
+console.log(person);
+
+const firstName = 'Jay';
+const lastName = 'Ghil';
+const fullName = firstName + ' ' + lastName;
```

Comment:
Hi Jay, you can access the first name and last name of your person object with dot notation.

```js
var fullName = person.firstName + ' ' + person.lastName;
```

Reply:
![prep-javascript-objects-vscode](https://user-images.githubusercontent.com/103128961/173695164-83d8da23-d380-4828-8e77-eecd427cd0a5.PNG)
![prep-javascript-objects-fixed](https://user-images.githubusercontent.com/103128961/173695166-c4533bf7-a667-423b-a52d-01b58a8ddcb9.PNG)

Reply:
Gotcha! I deleted lines 8 & 9. Also used dot notation for line 8 for fullName.